bus-mapping/src/evm/opcodes/balance.rs

```diff
@@ -43,3 +43,121 @@ impl Opcode for Balance {
         Ok(vec![exec_step])
     }
 }
+
+#[cfg(test)]
+mod balance_tests {
+    use super::*;
+    use crate::{
+        circuit_input_builder::ExecState,
+        mock::BlockData,
+        operation::{StackOp, RW},
+    };
+    use eth_types::{
+        bytecode,
+        evm_types::{OpcodeId, StackAddress},
+        geth_types::GethData,
+    };
+    use mock::eth;
+    use mock::test_ctx::{helpers::*, TestContext};
+    use pretty_assertions::assert_eq;
+
+    /// EVM behaviour
+    /// The BALANCE opcode pops address (20 bytes of data) off the stack
+    ///    * And pushes the balance of the corresponding account onto the stack.
+    ///    * If the given account doesn't exist (by checking non existing flag),
+    ///      then it will push 0 onto the stack instead.
```

Comment:
It's not required to explain the opcode functioning here. You can add small comments inside of the test if you're specifically checking something mentioned in these cases.

Reply:
Thanks for pointing it out. It's done